bot/bot.py

```diff
@@ -90,10 +83,11 @@ def error_usage_message(error=None):
 
 def usage_message():
 	usage_lines = ["**Usage:**",
-				   "`{} {}` Display help".format(TOKENS["prefix"], TOKENS["help"]),
-				   "`{} 3` Roll 3 dice default difficulty 6".format(TOKENS["prefix"]),
-				   "`{} 4{}8` Roll 4 dice difficulty 8".format(TOKENS["prefix"], TOKENS["roll_seperator"]),
-				   "`{} 5 {}` Roll 5 dice default difficulty 6 and exert".format(TOKENS["prefix"], TOKENS["exertion"]),
+				   "`{}{}` Display help".format(TOKENS["prefix"], TOKENS["help"]),
+				   "`{}3` Roll 3 dice default difficulty 6".format(TOKENS["prefix"]),
+				   "`{}4{}8` Roll 4 dice difficulty 8".format(TOKENS["prefix"], TOKENS["roll_seperator"]),
+				   "`{}5 {}` Roll 5 dice default difficulty 6 and exert".format(TOKENS["prefix"], TOKENS["exertion"]),
+				   "`{}5 {} to slay the dragon` Roll 5 dice default difficulty 6 and exert using an action label".format(TOKENS["prefix"], TOKENS["exertion"]),
```

Comment:
We should have a label example without exertion. Also, instead of `using an action label` say `labeling the roll "to slay the dragon"`. 

It might also be nice if the label example was more likely to be something that showed up in The Contract like `Fred initiative` or `sneaking around` or something.